tools/dts/zynqmp.dts

```diff
@@ -843,34 +843,6 @@
 			};
 		};
 
-		pcie@fd0e0000 {
```

Comment:
we try to keep the device trees as close to original as possible and use overlays to modify them. As an example see https://github.com/seL4/seL4/blob/master/src/plat/imx6/overlay-sabre.dts.

Reply:
So this may just be my system, but for whatever reason, the pcie device was generating devices with huge register addresses. So I just removed it for testing. 

Were you seeing that behavior at all?